packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -173,27 +180,47 @@ class DriveCommand extends RunCommandBase {
       'VM_SERVICE_URL': observatoryUri,
     };
 
-    sync_io.WebDriver driver;
+    async_io.WebDriver driver;
     // For web device, WebDriver session will be launched beforehand
     // so that FlutterDriver can reuse it.
     if (isWebPlatform) {
+      final Browser browser = _browserNameToEnum(
+          argResults['browser-name'].toString());
+      final String driverPort = argResults['driver-port'].toString();
       // start WebDriver
-      final Browser browser = _browserNameToEnum(argResults['browser-name'].toString());
-      driver = _createDriver(
-        argResults['driver-port'].toString(),
-        browser,
-        argResults['headless'].toString() == 'true',
-      );
+      try {
+        driver = await _createDriver(
+          driverPort,
+          browser,
+          argResults['headless'].toString() == 'true',
+        );
+      } catch (ex) {
```

Comment:
`on Exception` or other specific type. As is, this will catch NoSuchMethodError and TypeError

Reply:
Done.